packages/access-api/src/bindings.d.ts

```diff
@@ -42,7 +43,7 @@ export interface Env {
 
 export interface RouteContext {
   log: Logging
-  signer: Signer
```

Comment:
This `Signer` didn't resolve to anything before in my `tsc` setup, so was treated as `any`